Makefile

```diff
@@ -5,8 +5,9 @@ install:
 	python -m pip install -e .
 
 install-test:
-	python -m pip install -e ".[test]"
+	python -m pip install -e ".[test,spacy]"
```

Comment:
I noticed we would need to add these if we want to have any unit tests

Reply:
I see. I was surprised to see zero unit tests for this code, or any CI setup. Maybe something for another time.